pkg/bot/utils.go

```diff
@@ -27,3 +27,29 @@ import (
 func formatCodeBlock(msg string) string {
 	return fmt.Sprintf("```\n%s\n```", strings.TrimSpace(msg))
 }
+
+//AccessAndTypeCastToMap access and typecast of map object
+func AccessAndTypeCastToMap(key string, m map[string]interface{}) (map[string]interface{}, error) {
```

Comment:
Let's not export methods if not required
```suggestion
//accessAndTypeCastToMap access and typecast of map object
func accessAndTypeCastToMap(key string, m map[string]interface{}) (map[string]interface{}, error) {
```